firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Protocols/Chunks.cpp

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2020, the Pez Globo team and the Pufferfish project contributors
+ *
+ * Chunks.cpp
+ *
+ *  Created on: Nov 3, 2020
+ *      Author: Renji Panicker
+ *
+ * Unit tests to confirm behavior of chunk splitter
+ *
+ */
+
+#include "Pufferfish/Protocols/Chunks.h"
+
+#include "Pufferfish/Util/Vector.h"
+#include "catch2/catch.hpp"
+
+namespace PF = Pufferfish;
+
+SCENARIO(
+    "Protocols::ChunkSplitter behaves correctly across chunk boundary(256 bytes)", "[chunks]") {
+  GIVEN("A chunk splitter") {
```

Comment:
We could make this more specific by also describing the buffer size and data type (`uint8_t`, as opposed to `char`).

Reply:
updated as per suggestion.